doc/source/project/organization.rst

```diff
@@ -175,8 +175,23 @@ The Galaxy *committers* group may only commit directly to Galaxy (i.e. outside
 of a pull request and not following the procedures described here) the
 following two categories of patches:
```

Comment:
This text is inserted in the middle of a thought (two categories and then the list).